src/Compilers/CSharp/Portable/Symbols/ConstantValueUtils.cs

```diff
@@ -155,5 +157,21 @@ internal static string UnescapeInterpolatedStringLiteral(string s)
 
             return value;
         }
+
+        internal class CheckConstantInterpolatedStringValidity : BoundTreeWalkerWithStackGuardWithoutRecursionOnTheLeftOfBinaryOperator
```

Comment:
Should still be `sealed`

Reply:
I don't think so. It seems that sealed makes it inaccessible from some of the other files I'm calling it in.